### PR TITLE
update to work with ghc 7.10

### PR DESCRIPTION
**Pull request by codedmart**
@AtnNn I tested locally with 7.10 and works great. Also testing with travis-ci before opening this PR https://travis-ci.org/codedmart/haskell-rethinkdb/builds/58646640

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,10 +1,48 @@
...
 before_script:
     - source /etc/lsb-release && echo "deb http://download.rethinkdb.com/apt $DISTRIB_CODENAME main" | sudo tee /etc/apt/sources.list.d/rethinkdb.list
     - wget -qO- http://download.rethinkdb.com/apt/pubkey.gpg | sudo apt-key add -
     - sudo apt-get update -q
     - sudo apt-get install rethinkdb
     - rethinkdb --bind all &
+
+# Here starts the actual work to be performed for the package under test; any command which exits with a non-zero exit code causes the build to fail.
+script:
+ - if [ -f configure.ac ]; then autoreconf -i; fi
```
Comment (AtnNn) [inline .travis.yml]: I think this line is superfluous.

**Comment by AtnNn** [comment]
Very nice. Looks good to me execpt for:

* [ ] I want to test that the overlapping instances still work
* [ ] Some of the code in the travis file may be superfluous

File: .travis.yml (hunk 1 above)
Comment (codedmart) [inline .travis.yml]: OK, actually everything other then the before_script stuff is from here: https://github.com/hvr/multi-ghc-travis

**Comment by codedmart** [comment]
Do we not have tests that would show whether the overlapping instances work? I am fine removing any lines from the travis file you feel are superfluous. I was told on the haskell irc that https://github.com/hvr/multi-ghc-travis was the best to use.

**Comment by codedmart** [comment]
Closing merged in https://github.com/AtnNn/haskell-rethinkdb/commit/7450276b806dd4842a5e4c9eb4bdce320b380c84

**Pull request closed without merging**